utils/dc-chain/patches/newlib-4.3.0.20230120-kos.diff

```diff
@@ -243,374 +243,11 @@ diff --color -ruN newlib-4.3.0.20230120/newlib/libc/stdlib/assert.c newlib-4.3.0
  #ifndef HAVE_ASSERT_FUNC
  /* func can be NULL, in which case no function information is given.  */
  void
-@@ -72,3 +74,5 @@
+@@ -72,3 +74,7 @@
```

Comment:
This was in order to move the newlib patch extern out of syscalls.c, since that's no longer needed in the patch. Git's diff doesn't make it that clear.